ui/src/style/pdf-viewer.less

```diff
@@ -29,9 +29,8 @@
  * border styles. See the pdf.js code setting border styles here:
  * https://github.com/mozilla/pdf.js/blob/474fe1757e654e67d20a53774edb293aa073ce0b/src/display/annotation_layer.js#L19
  */
-.annotationLayer .linkAnnotation {
+.annotationLayer .linkAnnotation > a {
```

Comment:
Unless this is an unintended bug in the CSS overrides, a comment here would be useful so we know that this is necessary for enabling pdf.js's default link handling.

Reply:
Good question, the `display: none` was actually what was causing the click targets from pdf.js to go away, so this PR is removing the hack that was preventing the default link handling. The current comment that is here does actually apply well to the border override that we still want.

Given there isn't anything about native link handling involved anymore, I wasn't sure if it made sense to comment about it but interested in thoughts.

Reply:
Ah, I see, so we're removing something that was intentionally disabling it. Less important to me in this case, then.